### DISCUSSION TITLE
Maximum Throw Settings

### DISCUSSION THREAD

**Discussion by steveveach** [EdgeTX/edgetx#2716] [Q&A] [answered]
I have the RadioMaster TX16S Mark II running EdgeTX 2.7.1. I am confused by the maximum throw settings and was looking for some guidance. The particular model I'm trying to configure is using an Orange R620X v3 6-channel Telemetry Receiver using the DSMX protocol.

1. I see that in the transmitter Model Setup page there is a check-box labeled "Enable max. throw". When I check this box, I would have expected to be able to set the channel outputs to Min and Max values greater than 100 and -100, yet I cannot do this. They are fixed at a maximum of 100 and -100 with or without this check-box set.
2. I see that in the EdgeTX Companion, on the Setup tab for the model, there is no "Enable max. throw" checkbox. Instead there is an "Enable Max Throw" value box which is set to a value of 16. I did not set this, so I'm assuming this is a default value.
3. Lastly, I see that in the transmitter Model Setup, there are two other check-boxes labeled "Extended limits" and "Extended trims". What are these used for and how do they relate to the "Enable max. throw" check-box in the transmitter and the "Enable Max Throw" value of 16 in the EdgeTX Companion?

Thanks for any pointers or tips.

**Comment by richardclli** [comment, marked as answer]
Where did you set?  You should be able to do > 100 at output page.

**Comment by pfeerick** [comment]
The MPM "Enable max. throw" option will make it so that instead of having a 1100µs to 1900µs range output range, you will get 1024µs to 1976µs. This is output limiting that is done by the multi-protocol module, in order to make it behave like Spektrum - which reserves some servo travel space so that you can do things like subtrim without losing "full travel" as Spektrum shows it - [see more here](https://www.multi-module.org/using-the-module/protocol-details/dsm#method-1-enable-max-throw). So basically Enable Max throws just gives that extra 16% back right away, without needing to change any settings. 

![image](https://user-images.githubusercontent.com/5500713/201466325-e1374d7d-f296-46ef-b698-9cbcbb203f7a.png)

These two options are specific to edgetx/opentx. Regarding 1, extended limits, "Normal" output range is 988us to 2012us, extended limits lets you go further, from 732 to 2268us. This picture probably from [OTX University](http://open-txu.org/home/undergraduate-courses/basic-airplane-set-up-with-taranis-2/important-info-on-servo-travel/) explains that better:

![image](https://user-images.githubusercontent.com/5500713/201466754-1c132bd0-c554-478e-896b-f1c468f10a2b.png)

Extended limits are probably only of interest if you are doing specific mixes, as this is much more than the normal servo output range. 

Regarding Extended trims, normally the trim can only deflect the sticks by 25% - this allows for 100% - so it can basically override the stick completely. 

> I see that in the EdgeTX Companion, on the Setup tab for the model, there is no "Enable max. throw" checkbox. Instead there is an "Enable Max Throw" value box which is set to a value of 16. I did not set this, so I'm assuming this is a default value.

This seems to be a bug - it really should be a checkbox, as it is an on/off value. 

> I am confused by the maximum throw settings and was looking for some guidance.

Is there a particular reason you're after this option, or was it just general confusion as to what it does, and how it interacts with those other options?

**Reply by steveveach** [reply]
Thank you so much! This is a great explanation.

I'm looking to get just a little more throw on my aileron servos on my 3D plane. I'm at my mechanical limits with the control rods and horns and I can only get 40% deflection. I would like 45%. Just a small increase.

**Reply by pfeerick** [reply]
Ok, so Enable Max Throw should do what you want. If you reduce your outputs to 84%, you should be back to the original throws, and can then increase them as needed.

**Comment by steveveach** [comment]
pfeerick, I forgot to ask who should file the bug report about the value field instead of a checkbox in EdgeTX Companion? I don't know how to do that.

"I see that in the EdgeTX Companion, on the Setup tab for the model, there is no Enable max. throw checkbox. Instead there is an Enable Max Throw value box which is set to a value of 16. I did not set this, so I'm assuming this is a default value."

"This seems to be a bug - it really should be a checkbox, as it is an on/off value."

**Reply by pfeerick** [reply]
Just reported it here #2723. Thanks :)

**Comment by steveveach** [comment]
Thanks!

*Steve Veach*
Laguna Woods, CA
Cell: 773-704-9661
http://steveveach.com

**Comment by steveveach** [comment]
Thanks. It worked.

*Steve Veach*
Laguna Woods, CA
Cell: 773-704-9661
http://steveveach.com
